src/Tribe/Commerce/PayPal/Main.php

```diff
@@ -2439,15 +2440,29 @@ public function attendee_decreases_inventory( array $attendee ) {
 		 */
 		$ignore_pending = apply_filters( 'tribe_tickets_tpp_pending_stock_ignore', $this->ignore_pending_stock_logic );
 
+		$purchase_time = false;
+		$order         = false;
+
 		if (
 			'on-pending' === tribe_get_option( 'ticket-paypal-stock-handling', 'on-complete' )
 			&& ! $ignore_pending
 			&& Tribe__Tickets__Commerce__PayPal__Stati::$pending === $order_status
 			&& false !== $order_id
-			&& false !== $order = Tribe__Tickets__Commerce__PayPal__Order::from_attendee_id( $order_id )
 		) {
-			/** @var \Tribe__Tickets__Commerce__PayPal__Order $order */
-			$order_creation_timestamp = Tribe__Date_Utils::wp_strtotime( $order->get_creation_date() );
+			$purchase_time = Tribe__Utils__Array::get( $attendee, 'purchase_time', false );
+
+			if ( false !== $order = Tribe__Tickets__Commerce__PayPal__Order::from_attendee_id( $attendee_id ) ) {
+				/** @var \Tribe__Tickets__Commerce__PayPal__Order $order */
```

Comment:
:no_entry_sign: Missing short description in doc comment

<sub>hash: 30fc64ed29b023ba316ce7aac831bbbe</sub>